### DISCUSSION TITLE
cannot connect to sqlexpress with windows authentication

### DISCUSSION THREAD

**Discussion by heradv** [dbeaver/dbeaver#14744] [Q&A]
i can not connect to sqlexpress with windows authentication setting.  i can connect to it with SSMS with the same settings. how should i do?

**Comment by LonwoLonwo** [comment]
Any errors in the [Error log](https://github.com/dbeaver/dbeaver/wiki/Log-files)?

Did you try to connect this way?

![2021-12-02 18_09_59-DBeaver Ultimate 21 3 0 - users](https://user-images.githubusercontent.com/45152336/144448985-c8fbef27-6e09-4217-98d8-1807f9d15533.png)

![2021-12-02 18_10_24-DBeaver Ultimate 21 3 0 - users](https://user-images.githubusercontent.com/45152336/144448988-1a20364d-3819-4845-9332-dbee376febc3.png)

**Reply by heradv** [reply]
yes, i tried this way and got an error window with connection timeout error text. though i am not sure if tcp port can be enable in express version at all, but there are no option in DBeaver to connect by server name as in SSMS.

**Reply by LonwoLonwo** [reply]
Helo @heradv 
You can use "SQL Server authentication"

![2021-12-21 13_03_21-Window](https://user-images.githubusercontent.com/45152336/146911261-43572d50-1928-4b6f-a430-6da188aa4848.png)
